src/App.tsx

```diff
@@ -1,26 +1,247 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useRef, useState } from 'react';
+// import { UserWarning } from './UserWarning';
+import { deleteTodos, getTodos, postTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { USER_ID } from './api/todos';
+import { ErrorMessange } from './component/ErrorMessange';
+import { Statys } from './types/statys';
+import { TodoList } from './component/TodoList';
+import { Footer } from './component/Footer';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  // if (!USER_ID) {
+  //   return <UserWarning />;
+  // }
+
+  const [title, setTitle] = useState('');
+  const [error, setError] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<Statys>(Statys.ALL);
+  const [isLoading, setIsLoading] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [itemLeft, setItemLeft] = useState(0);
+  const [loadingTodoId, setLoadingTodoId] = useState<number | null>(null);
+
+  function loadTodos() {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
  const loadTodos = () => {
```